### PR TITLE
Solid FAQ

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
There are a few questions on the forum and in response to talks on Solid around privacy and Solid so I propose we try and summarise the essence of the questions in an FAQ along with an answer that addresses the key points. 

In this pull request, there's a very rough jotted down version of an FAQ to kick off the draft. 

In particular, I would like to invite the specification editors to refine this copy.

### CODE REVIEW

File: pages/faqs.md [hunk 1]
```diff
@@ -170,6 +170,10 @@ The Semantic Web and Linked Data technologies are centered on the question of in
 
 <a id="fewer_passwords"/>
 
+## Are all Solid compatible Pod providers and apps privacy preserving? 
+
+When software services are Solid compatible it means that you as a user are free to switch to a similar alternative and bring your data along. For example, if you wanted to switch messaging app you could bring your contacts and chat history with you to the new messaging app.
```
Comment (RubenVerborgh) [inline pages/faqs.md:175]: That's a non-answer though; it answers a different question than the one asked.

That said, "privacy-preserving" is a notion that requires an immense burden of proof, what we cannot guarantee in general (just like not every website on the Web if safe).

**Comment by Mitzi-Laszlo** [comment]
Feel free to change both question and answer phrasing. The important element to include is clarity around the interaction between Solid and privacy. If this FAQ doesn't explain everything about that interaction that is ok. If there are misunderstandings later on we can always adapt and elaborate to clarify and improve.

**Comment by RubenVerborgh** [comment]
I don't think this one is ready for review yet; no changes since last time.

**Comment by Mitzi-Laszlo** [comment]
Indeed, needs technical input.

**Comment by RubenVerborgh** [comment]
> Indeed, needs technical input.

The usual way this works on GitHub is:
- put the PR in draft (indicating that is not ready)
- assign to a person who will do the work
- have them work on a solution until they are happy
- mark the PR as "ready for review" (which puts it out of draft)
- assign reviewers
- await reviews
- merge (or go back to draft)

So that's why I was confused by `Mitzi-Laszlo requested a review from solid/editors 22 hours ago`, because the PR is not ready for review yet. You probably want to assign instead.

**Comment by Mitzi-Laszlo** [comment]
The blocker to creating an FAQ addressing Solid and privacy is technical input.

[Here's an issue](https://github.com/solid/solidproject.org/issues/286) which is now included in the solidproject.org kanban board column called "to do once there are technical resources". 

I'll close this pull request and we can collectively work on the issue instead.

**Pull request closed without merging**